File: restore_css.py
import subprocess

try:
    with open('restored_timeline.css', 'w', encoding='utf-8') as f:
        subprocess.run(['git', 'show', 'HEAD:components/Timeline.module.css'], stdout=f, check=True)
    print("Successfully restored file.")
except Exception as e:
    print(f"Error: {e}")

File: restore_css.js
const { execSync } = require('child_process');
const fs = require('fs');
try {
    const output = execSync('git show HEAD:components/Timeline.module.css', { encoding: 'utf8' });
    fs.writeFileSync('restored_timeline.css', output);
    console.log('Successfully restored file.');
} catch (e) {
    console.error(e);
}
